modules/OCPPConfiguration/main/event_handler.hpp

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#pragma once
+
+#include "generated/types/ocpp.hpp"
+#include "mapping_reader.hpp"
+
+namespace module {
+
+class EventHandler {
```

Comment:
Please add a brief description what this class does

Reply:
Done